pghoard.spec

```diff
@@ -24,11 +25,17 @@ Support for Microsoft Azure is experimental.
 %setup -q -n pghoard
 
 
+%build
+go version && go build golang/pghoard_postgres_command_go.go
+
+
 %install
+test -f pghoard_postgres_command_go && sed -e s,pghoard_postgres_command,pghoard_postgres_command_go,g -i pghoard/restore.py
```

Comment:
Looks like this breaks unit tests because of a hack on `test/test_basebackup.py` line 368.  I'll figure out something for it.